docs/ops/activation/SoftPlus_4.md

```diff
@@ -0,0 +1,45 @@
+## SoftPlus <a name="SoftPlus"></a>
```

Comment:
@dmitry-gorokhov, I see that in MKLDNN plugin the operation is called "Softplus" (lower case "p"). Could you adjust the name to the specification? MO never emitted IR with operation "Softplus" so the change will not break backward compatibility.